backend/docker-compose.yaml

```diff
@@ -0,0 +1,50 @@
+version: "3"
+
+# Configure postgres database using, for example, postgres:16 image from docker hub.
+# Run with "docker compose up"
+
+services:
+  postgres:
+    image: postgres:alpine
+    container_name: velato-postgres
+    environment:
+      POSTGRES_DB: velato_db
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: admin
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+      - ./init-db.sql:/docker-entrypoint-initdb.d/init-db.sql
```

Comment:
Make sure you have the script (./init-db.sql) otherwise you can comment this line.